CSSGapDecorations/explainer.md

```diff
@@ -397,65 +397,111 @@ properties. See [CSSWG Issue
 
 ### Placement of gap decorations
 
-Allow authors to specify where gap decorations start and end within a container.
+An author may want to apply different sets of gap decorations to different
+regions of a given container layout. We refer to such regions as a *gap
+decoration areas*.
 
-An author may specify more than one such region and apply a different set of gap
-decorations to each. Within this document, we refer to such a region as a *gap
-decoration area*. Much like CSS Transitions and Animations, all gap decoration
-properties may take a comma-delimited list of values. Each entry in such a list
-is applied to the corresponding entry in the list of gap decoration areas. If a
-given property's list length is shorter than the gap decoration area list
-length, the shorter list cycles back to the beginning as needed.
+The author defines these areas using the `rule-areas` property. In grid and
+masonry containers, these areas are defined using tuples of line names or numbers,
+similar to the `grid-area` property, each of which is assigned a name:
 
-Gap decoration area properties are defined per container type.
+```css
+  rule-areas: --first-row 1 / 1 / 2 / -1, --first-column 1 / 1 / -1 / 2;
+```
+
+Then, on other gap decoration properties such as `*-rule-width`, `*-rule-style`,
+and `*-rule-color`, the author can then specify first a "default" set of values
+for the container, then a named area, then a set of values that applies to that
+area, and so on:
 
-#### Grid
+```css
+  rule: 1px solid black, 1px solid gray [--first-row] 3px solid black [--first-column] 1px solid blue;
```

Comment:
Same here, an explanation of the example would be helpful on how these rules end up applying.

Also, what happens if the named area and rules associated have a different number of gaps? I assume the cycling and cutoff behavior works the same here? If so, would be good to note that, as well, maybe with an example

Reply:
Added explanatory text and expanded the example a bit to clarify.